src/python/pants/backend/shell/target_types.py

```diff
@@ -226,7 +226,9 @@ class ShellCommandToolsField(StringSequenceField):
     required = True
     help = (
         "Specify required executable tools that might be used.\n\n"
-        "Only the tools explicitly provided will be available on the search PATH."
+        "Only the tools explicitly provided will be available on the search PATH, "
+        "and these tools must be found on the paths provided by "
+        "[shell-setup].executable-search-paths (which defaults to the system PATH)."
```

Comment:
Love it. Great detail!

Reply:
I am so excited for this!

Random nit: Don't option names in config have to use underscores? i.e., `executable_search_paths`? We let them be with dashes on the cmd line (`--shell-setup-executable-search-paths`) because `--shell-setup-executable_search_paths` looks awful, but I think we at least encourage underscores in option names (but dashes in scope names, so it's still `shell-setup`).

Reply:
Oh yeah true, I think it needs to be underscores for the option part